docker-compose-run

```diff
@@ -1,18 +1,7 @@
-#!/usr/bin/env python3
+#!/bin/bash
 
-import getpass
-import os
-import sys
-import json
-import subprocess
-import urllib.parse
-
-os.environ["USER_NAME"] = getpass.getuser()
-os.environ["USER_ID"] = str(os.getuid())
-os.environ["GROUP_ID"] = str(os.getgid())
-os.environ["RUN"] = urllib.parse.quote(json.dumps(sys.argv[1:]))
-
-try:
-    subprocess.check_call(["docker-compose", "up", "--exit-code-from=build"])
-except subprocess.CalledProcessError as e:
-    exit(e.returncode)
+docker-compose run \
+    -e USER_NAME=$USER \
+    -e USER_ID=`id -u $USER` \
+    -e GROUP_ID=`id -g $USER` \
+    build $@
```

Comment:
Why converting that in bash?
are you sure the the id command works on Windows?
Are you sure the the $@ handle correctly the arguments?

Reply:
> Why converting that in bash?
because i found it really simpler.

> are you sure the the id command works on Windows?
yes, I missed this.

> Are you sure the the $@ handle correctly the arguments?
yes, I'm sure in bash, but bash won't work on windows.

Everything here should be in the makefile.

Reply:
??? We don't want  a secound makefile ...